SystemServices/System.json

```diff
@@ -1784,6 +1787,9 @@
         },
         "setTimeZoneDST":{
             "summary": "Sets the system time zone. See `getTimeZones` to get a list of available timezones on the system.",
+			"events":{
+                "onTimeZoneDSTChanged": "Triggered when device time zone DST changed"
```

Comment:
Change description to "Triggered when device time zone changed" to match the one in the event details below.